.goreleaser.yml

```
@@ -0,0 +1,42 @@
+before:
+  hooks:
+    - go mod download
+builds:
+  -
+    ldflags:
+      - -s -w -X "github.com/sonatype-nexus-community/nancy/buildversion.BuildVersion={{.Version}}"
+      - -s -w -X "github.com/sonatype-nexus-community/nancy/buildversion.BuildTime={{time "2006-01-02T15:04:05Z07:00"}}"
+      - -s -w -X "github.com/sonatype-nexus-community/nancy/buildversion.BuildCommit={{.FullCommit}}"
```

The default value is a one liner, https://goreleaser.com/customization/#Builds

```suggestion
      - >-
        -s -w
        -X "github.com/sonatype-nexus-community/nancy/buildversion.BuildVersion={{.Version}}"
        -X "github.com/sonatype-nexus-community/nancy/buildversion.BuildTime={{time "2006-01-02T15:04:05Z07:00"}}"
        -X "github.com/sonatype-nexus-community/nancy/buildversion.BuildCommit={{.FullCommit}}"
```

---

I'm not ANTI this, but I kinda like it as three separate commands (since it's doing it three different places). I don't have enough experience with goreleaser to know if one is preferable to the other. My main concern would be if one fails, etc...

---

The readability is I THINK easier with it broken out like this, is my main pro to having them the way I have it.

---

But yours isn't bad for that either (it's just clear on the original that it's running -s -w -X on all three, at a quick glance)